### DISCUSSION TITLE
Kata 4  ¿Datos de marte duplicados?

### DISCUSSION THREAD

**Discussion by EnriqueTow** [LaunchX-InnovaccionVirtual/CursoIntroPython#202] [Q&A] [answered]
Saludos, espero me pueden ayudar, pues resulta que en la parte 2 de la kata 4 me piden meter los siguientes datos-
![image](https://user-images.githubusercontent.com/99060536/153724864-c2ea0842-48eb-4624-8571-a9aa2af1937f.png)
Hasta ahí todo entendible pero mas adelante me pide que meta otros datos para el "template" pero los datos que me dan son los mismos que en la imagen anterior
![image](https://user-images.githubusercontent.com/99060536/153725601-0bbe3516-823a-49bb-9174-52119e556596.png)
¿Lo dejó así? o pasé algo por alto

**Comment by martinGM05** [comment, marked as answer]
Emm no hay nada mal, el punto del ejercicio es entender como funcionan las cadenas.

**Reply by EnriqueTow** [reply]
O sea que en las dos partes uso marte, su gravedad y nombre para el "template"? Solo es la duda de dejarlo así, por que no se si se vaya a calificar y me lo pongan mal

**Reply by DiegoPenalozaJuarez** [reply]
Debes usar en la primera parte los datos de la tierra, y en la segunda los de marte para entender que el template no toma los segundos datos. Despues deberas realizar un nuevo template con los datos de marte para que te funcione. AL final debes tener dos templates para cada planeta.

**Reply by EnriqueTow** [reply]
Ahh ya, ya vi. Gracias creo que habia editado los datos por default que te daban de la luna
